backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/greeting/GreetingRepository.java

```diff
@@ -91,6 +102,22 @@ public Greeting createGreeting(@NonNull final CreateGreetingRequest request)
 		return fromRecord(record);
 	}
 
+	@NonNull
+	public Greeting save(@NonNull final Greeting greeting)
+	{
+		final I_C_Greeting record = InterfaceWrapperHelper.loadOrNew(greeting.getId(), I_C_Greeting.class);
+
+		record.setName(greeting.getName());
+		record.setGreeting(greeting.getGreeting().translate(Env.getAD_Language()));
```

Comment:
hmm, here we shall use the base language
and then... we shall update C_Greeting_Trl(s) too

Reply:
... or consider introducing a method createOrUpdate(GreetingUpsertRequest)